Jenkinsfile

```diff
@@ -7,18 +7,23 @@ node {
       checkout scm
     }
 
+    env.AWS_DEFAULT_REGION = 'us-east-1'
+    env.RF_SETTINGS_BUCKET = 'rasterfoundry-staging-config-us-east-1'
+    env.RF_ARTIFACTS_BUCKET = 'rasterfoundry-global-artifacts-us-east-1'
+
     // Execute `cibuild` wrapped within a plugin that translates
     // ANSI color codes to something that renders inside the Jenkins
     // console.
     stage('cibuild') {
+
+      // Override Settings bucket for testing
+      env.RF_SETTINGS_BUCKET = 'rasterfoundry-testing-config-us-east-1'
```

Comment:
A little worried setting `RF_SETTINGS_BUCKET` here will leave it set outside of the `cibuild` stage. Might be worth moving just the assignment to `rasterfoundry-staging-config-us-east-1` before the `cipublish` stage.